### DISCUSSION TITLE
Invitations of a company end of year event

### DISCUSSION THREAD

**Discussion by rhonalindiefolks** [pretix/pretix#3655] [Q&A] [answered]
Hello,

I would like to ask you if there is any recommended flow in order to use pretix to develop a product to manage the invitations of a company end of year event. The use case would be the company uploading an Excel file with the full name and email address of the guests, and send RSVP emails with a YES/NO confirmation button, and later generate the tickets and send them through email with the QR code as a pdf and a pkpass for apple wallet.

Any recommendations would be greatly appreciated!!

**Comment by raphaelm** [comment, marked as answer]
pretix does not really have a great RSVP flow with yes/no options

**Reply by rhonalindiefolks** [reply]
Thank you
